kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -2638,15 +2657,24 @@ def run_fencer(self, ps_uuid, created_time):
             logger.debug('setup fencer for ps: %s, create time: %d' % (ps_uuid, created_time))
             self.run_fencer_timestamp[ps_uuid] = created_time
 
+        if origin_uuid is not None:
+            self.fencer_storage_list.add(origin_uuid)
+        else:
+            self.fencer_storage_list.add(ps_uuid)
+
+
     def cancel_fencer(self, ps_uuid):
         with self.fencer_lock:
             for key in self.run_fencer_timestamp.keys():
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)